filters/encrypt/README.md

```diff
@@ -0,0 +1,83 @@
+# encrypt package
+
+The encrypt package implements a new Filter that supports filtering fields in an
+event payload using a custom tag named `classification`.
+
+**tagged struct example**
+```go
+type testPayloadStruct struct {
+    Public    string `classified:"public"`
+	Sensitive string `classified:"sensitive"`
+    Secret    []byte `classified:"secret"`
+}
+
+```
+
+encrypt.Filter supports filtering the following struct field types within an
+event payload, when they are tagged with a `classification` tag:
```

Comment:
s/classification/classified?

Reply:
BTW while this is inconsistent, I sort of think `classification` is the _right_ thing, although it's much wordier to deal with than `classified` :-/  But curious if others agree.

Reply:
ty!